src/modules/graphics/graphics.c

```diff
@@ -1412,7 +1431,7 @@ Shader* lovrShaderCreate(ShaderInfo* info) {
     spv[i].attributes = tempAlloc(spv[i].attributeCount * sizeof(spv_attribute));
     spv[i].resources = tempAlloc(spv[i].resourceCount * sizeof(spv_resource));
 
-    result = spv_parse(info->stages[i]->data, info->stages[i]->size, &spv[i]);
+    result = spv_parse(info->stages[i]->data, (uint32_t)info->stages[i]->size, &spv[i]);
```

Comment:
Hey question, you call spv_parse on what appears to be the exact same data twice in this function. Why?

Reply:
First call returns the amount of memory to allocate